src/keyring/KeyringController.test.ts

```diff
@@ -155,10 +155,23 @@ describe('KeyringController', () => {
     } catch (e) {
       error2 = e;
     }
+
+    let error3;
```

Comment:
What do you think about using `expect(() => { ... }).toThrow(...)` for these three errors? I think that might be more idiomatic in this case than try/catch.

Reply:
Yeah that makes sense to me. I'll make that change!

Reply:
done here https://github.com/MetaMask/controllers/pull/739/commits/6ddceb2202067fc3d932ca7335b1668573f39957 @mcmire